about.html

```diff
@@ -0,0 +1,90 @@
+                <img class="logo" src="./assets/img/logo.png" alt="Group logo">
+                <img class="close-btn" src="assets/img/close-icon.png" alt="">
+                <nav class="desktop-menu">
+                    <ul>
+                        <li class="nav-item"><a class="menu-item" href="./index.html">Home</a></li>
+                        <li class="nav-item"><a class="menu-item" href="#Program">Program</a></li>
+                        <li class="nav-item"><a class="menu-item" href="#">Join</a></li>
+                        <li class="nav-item"><a class="menu-item" href="#">Sponsor</a></li>
+                        <li class="nav-item"><a class="menu-item" href="#">News</a></li>
+                        <li class="nav-item1"><a class="concert" href="#"></a>Spring Day World Tour</li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <!------------------ ABOUT ------------------>
+        <section class="about-section">
+            <div class="about-section-content">
+                <h1>"Hello! Armys"</h1>
+                <h2>SPRING DAY WORLD TOUR 2025</h2>
+                <p class="p">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Maiores cumque eius rerum velit eaque, ex, optio alias dolor necessitatibus, repellendus distinctio vero dignissimos temporibus incidunt. Aut temporibus maxime perspiciatis nulla ex ut vitae labore fugiat fuga. Modi, recusandae.</p>
+                <p>For more informations about The World Tour<br>Please contact us via email<br><a href="">support@weverse.io</a></p>
+            </div>
+        </section>
+        <section class="about-concerts">
+            <h2>봄날 월드 투어 로고</br>SPRING DAY World Tour Logo</h2>
+            <hr>
+            <p class="mobile">The Logo of BTS Spring Day Wolrd Tour 2025 was decided through an international competion from 8.February to 7.March</p>
+            <p class="desktop">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Harum quod dignissimos eligendi quas eos aut ipsam odit, hic qui a laborum veritatis aperiam necessitatibus quisquam iure debitis ullam. Qui, sed amet totam voluptate, possimus obcaecati aperiam eligendi explicabo aliquam quisquam quibusdam similique distinctio, animi itaque fugit? Consequatur vel placeat tempora fugit!</p>
+            <img class="logo-img" src="./assets/img/spring-day.png" alt="">
+            <div class="hr-normal"></div>
+            <h2>See the past Wolrd Tours</h2>
+            <hr>
+            <p>Take a look at the past two world tours that took place in several cities across Asia, US and Europe.</p>
+
+            <div class="past-events">
+                <div class="past-event">
+                    <img src="./assets/img/Wings-Tour.png" alt="">
+                </div>
+
+                <div class="past-event">
+                    <img src="./assets/img/LY-Tour.png" alt="">
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Good job but your about page design is a bit different from the original design :cry:  You can check the difference with the screenshot I'm sharing with you below :arrow_down:  Please make sure to use the exact same colors, for text, backgrounds, borders, etc, from the original design as required for this project 👍

![Capture](https://user-images.githubusercontent.com/6615394/225180485-0f851d60-585b-4674-89c1-80ffa9b0d4e6.PNG)